### PR TITLE
Update resume-review.md

### PR DESCRIPTION
**Pull request by chryzcodez**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/resume-review.md [hunk 1]
```diff
@@ -17,4 +17,4 @@ Items we can review on a live stream:
 
 You can also join the discord community [here](https://discord.com/invite/jZQs6Wu)
 
-Feel free to check out other cool repositories of Eddie Jaoude Community [here](https://github.com/EddieJaoudeCommunity)
+Feel free to check out other cool repositories of EddieHub Community [here](https://github.com/EddieHubCommunity)
```
Comment (SashoStoichkov) [inline .github/ISSUE_TEMPLATE/resume-review.md:20]: Here it should be `EddieHub` only

**Review by SashoStoichkov — CHANGES_REQUESTED** [review]
Looks good 👍🏻

File: .github/ISSUE_TEMPLATE/resume-review.md (hunk 1 above)
Comment (schmelto) [reply .github/ISSUE_TEMPLATE/resume-review.md:20]: No that is correct

**Review by schmelto — APPROVED** [review]
Lgtm

**Comment by AllanRegush** [comment]
@eddiejaoude Should we use EddieHub or EddieHubCommunity?

**Comment by eddiejaoude** [comment]
Good questions. I am on the fence. Either way, it is better than what it is now. We can always tweak it again if we need. Currently it is definitely wrong with the invalid name and url.

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Pull request merged**